portal-frontend/src/app/features/notice-of-intents/notice-of-intent-details/additional-information/additional-information.component.html

```diff
@@ -35,6 +35,19 @@
         <app-no-data [showRequired]="showErrors"></app-no-data>
       </div>
     </div>
+    <ng-container *ngIf="isMobile">
+      <ng-container *ngFor="let structure of _noiSubmission.soilProposedStructures; let i = index">
+        <div class="mobile-structure-card" [class.mobile-structure-card-border]="i > 0">
```

Comment:
I think we have two options here: 

- Remove the unnecessary code from the mobile component (isReviewStep) and proceed with current implementation here.
- Add the 'isReviewStep' back, move the mobile component to the shared folder, and reuse the component here that matches with our previous implementations.

Reply:
I removed the isReviewStep property, since the designs were different.